pkgs/applications/video/mpv/default.nix

```diff
@@ -55,11 +54,13 @@
 , cacaSupport        ? true,           libcaca
 , cmsSupport         ? true,           lcms2
 , dvdnavSupport      ? stdenv.isLinux, libdvdnav
+, dvdbinSupport      ? stdenv.isLinux
 , jackaudioSupport   ? false,          libjack2
 , javascriptSupport  ? true,           mujs
 , libpngSupport      ? true,           libpng
 , openalSupport      ? true,           openalSoft
 , pulseSupport       ? config.pulseaudio or stdenv.isLinux, libpulseaudio
+, pipewireSupport    ? config.pulseaudio or stdenv.isLinux, pipewire
```

Comment:
Why is pipewire support conditional on *pulseaudio*? Pipewire does allow pulseaudio apps via the fake pulse server but it does not work the other way around.

Reply:
Should be just `stdenv.isLinux`.

Reply:
> Why is pipewire support conditional on _pulseaudio_?

because I did a copy paste error. Simple as that.

Reply:
No worries! I was just wondering if I'd missed some implications or side effects that pulseaudio might have had.